Lesson_10_Algorithm/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        int[] array = { 4, 9, 7, 6, 2, 3, 8, 5, 0, 1 };
+
+        SortMathod(array);
+
+        PrintArray(array);
+    }
+
+    private static void SortMathod(int[] array)
+    {
+        for (int i = 0; i < array.Length - 1; i++)
+        {
+            for (int j = i + 1; j < array.Length; j++)
+            {
+                if (array[i] > array[j])
```

Comment:
introduce a variable for `array[i]` and move out it from this loop

Reply:
This will help to save time on getting a value using its index

Reply:
done!

Reply:
done!